boards/samr21-xpro/include/board.h

```diff
@@ -57,6 +59,25 @@ extern "C" {
 #define AT86RF2XX_PARAM_SLEEP      GPIO_PIN(PA, 20)
 #define AT86RF2XX_PARAM_RESET      GPIO_PIN(PB, 15)
 
+/**
+ * @brief    EDBG provides a EUI-64, the same that is printed on the board
+ */
+static inline int _edbg_get_eui64(const void *arg, eui64_t *addr)
+{
+    (void) arg;
+    return edbg_get_eui64(addr);
+}
+
+/**
+ * @name    EUI sources on the board
+ *          EUI-64 inside EDBG for the internal radio
+ * @{
+ */
+#define EUI64_PROVIDER_FUNC   _edbg_get_eui64
+#define EUI64_PROVIDER_TYPE   NETDEV_AT86RF2XX
+#define EUI64_PROVIDER_INDEX  0
+/** @} */
+
```

Comment:
This should work for other `BOARD`s as well right?

Reply:
On `same54-xpro` the EUI-48 for the Ethernet interface is provided via a at24mac EEPROM.
edbg does not always provide an EUI.